instrumentation-core/servlet-2.2/src/main/java/io/opentelemetry/instrumentation/servlet/ServletHttpServerTracer.java

```diff
@@ -141,8 +149,15 @@ private static String getSpanName(HttpServletRequest request) {
   public void updateServerSpanNameOnce(Context attachedContext, HttpServletRequest request) {
     if (AppServerBridge.isPresent(attachedContext)
         && !AppServerBridge.isServerSpanNameUpdatedFromServlet(attachedContext)) {
-      getServerSpan(request).updateName(getSpanName(request));
+      updateSpanName(request);
       AppServerBridge.setServletUpdatedServerSpanName(attachedContext, true);
     }
   }
+
+  public void updateSpanName(HttpServletRequest request) {
+    Span span = getServerSpan(request);
+    if (span != null) {
```

Comment:
It is possible when liberty request integration didn't start span because the method from where it is supposed to start it wasn't called, but the integration that is supposed to update span name is called. I'll change it so that it won't try to update span when it wasn't started.